### PR TITLE
feat(eslint): Added spaced-before-blocks rule

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
### Changes

- Added two rules 1. `space-before-blocks` and 2. `keyword-spacing`<ONE>
- Fixed some errors using `npm run lint:fix` that occurred by running linting tests<TWO>

Documentation Rules : 
- [space-before-blocks](https://eslint.org/docs/rules/space-before-blocks)
- [keyword-spacing](https://eslint.org/docs/rules/keyword-spacing)

### Flags

- I had also updated eslint command patterns see #47 for more details if you think this is wrong or inappropriate I will re-update my PR for this
Edit: now these changes are made in #51 <ONE>

### Related Issues
- Issue #<NUMBER>
- Pull Request #44  see this [comment](https://github.com/accordproject/cicero-word-add-in/pull/44#discussion_r617909879)<NUMBER>

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "build:dev": "webpack --mode development --https false",
     "build-dev": "webpack --mode development --https false && echo . && echo . && echo . && echo Please use 'build:dev' instead of 'build-dev'.",
     "dev-server": "webpack-dev-server --mode development",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint:fix": "eslint '**/*.{js,jsx}' --fix",
+    "lint": "eslint **/*.{js,jsx}",
```
Comment (algomaster99) [inline package.json]: This is not related to the PR so it is better to make a separate PR for it.

File: .eslintrc.json [hunk 2]
```diff
@@ -38,7 +38,9 @@
           "ArrowFunctionExpression": true,
           "FunctionExpression": true
         }
-    }]
+    }],
+    "space-before-blocks":["error", "always"],
+    "keyword-spacing": ["error", {"after":true}]
```
Comment (algomaster99) [inline .eslintrc.json]: `["error", { "after": true }]`

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Dropped a few changes. Rest everything looks good.

File: package.json (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply package.json]: @algomaster99 Sure!, I will reopen #47 and make these changes there :)

**Comment by Aniruddha-Shriwant** [comment]
@algomaster99 I had updated my PR according to the requested changes

File: package.json (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply package.json]: I had made these changes in #51

File: .eslintrc.json [hunk 3]
```diff
@@ -38,7 +38,9 @@
           "ArrowFunctionExpression": true,
           "FunctionExpression": true
         }
-    }]
+    }],
+    "space-before-blocks":["error", "always"],
```
Comment (algomaster99) [inline .eslintrc.json]: Just add a space after `:`

`"space-before-blocks": ["error", "always"],`

Maybe we can incorporate formatting of this configuration in `npm run lint`. But that is a task for a different PR.

**Review by algomaster99 — APPROVED** [review]
Looks good to me! Dropped one change.

File: .eslintrc.json (hunk 3 above)
Comment (Aniruddha-Shriwant) [reply .eslintrc.json]: > Just add a space after `:`

Okay sure I will update my PR 

> But that is a task for a different PR.

I will open a PR for this too

**Review by algomaster99 — APPROVED** [review]
Nice work 🎉
